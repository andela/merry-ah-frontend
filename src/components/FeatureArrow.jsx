import React from 'react';
import PropTypes from 'prop-types';

const Arrow = ({ direction, clickFunction }) => {
  return (<div className='slide-arrow' onClick={clickFunction}>
    <i className={`fa fa-caret-${direction} fa-5x`}></i>
</div>);
};

Arrow.propTypes = {
  direction: PropTypes.string.isRequired,
  clickFunction: PropTypes.func.isRequired,
};

export default Arrow;
