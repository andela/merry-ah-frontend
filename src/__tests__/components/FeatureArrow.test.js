import React from 'react';
import { shallow } from 'enzyme';
import FeatureArrow from '../../components/FeatureArrow.jsx';
import props from '../__mocks__/__featureMocks__/arrowMock';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<FeatureArrow {...props} />);
});
describe('FeatureArrow Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
