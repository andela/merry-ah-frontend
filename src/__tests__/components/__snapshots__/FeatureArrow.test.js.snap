// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureArrow Component should match snapshot 1`] = `
<div
  className="slide-arrow right"
  onClick={[Function]}
>
  &#9654;
</div>
`;
